src/Arcus.BackgroundJobs.AzureActiveDirectory/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -15,12 +16,32 @@ public static class IServiceCollectionExtensions
         /// Adds the <see cref="ClientSecretExpirationJob"/> scheduled job
         /// which will query Azure Active Directory for applications that have expired or soon to be expired secrets and send a CloudEvent to an Event Grid Topic.
         /// </summary>
+        /// <remarks>
+        ///     Make sure that you register an <see cref="IEventGridPublisher"/> instance that the background job can use to publish events for potential expired Azure Application secrets.
+        ///     For more information on Azure EventGrid, see: <a href="https://eventgrid.arcus-azure.net/Features/publishing-events" />.
+        /// </remarks>
+        /// <param name="services">The services to add the background job to.</param>
+        /// <exception cref="ArgumentNullException">Thrown when <paramref name="services"/> is <c>null</c>.</exception>
+        public static IServiceCollection AddClientSecretExpirationJob(this IServiceCollection services)
+        {
+            Guard.NotNull(services, nameof(services));
+            return AddClientSecretExpirationJob(services, configureOptions: null);
+        }
+
+        /// <summary>
+        /// Adds the <see cref="ClientSecretExpirationJob"/> scheduled job
+        /// which will query Azure Active Directory for applications that have expired or soon to be expired secrets and send a CloudEvent to an Event Grid Topic.
+        /// </summary>
+        /// <remarks>
+        ///     Make sure that you register an <see cref="IEventGridPublisher"/> instance that the background job can use to publish events for potential expired Azure Application secrets.
+        ///     For more information on Azure EventGrid, see: <a href="https://eventgrid.arcus-azure.net/Features/publishing-events" />.
```

Comment:
I'm also wondering if we can improve our background job registration by initiating the job itself internally and throwing our own kind of exception to tell that there's missing an event grid publisher, and pointing towards this URL. Seems that this could also be an improvement on other jobs as well.

Reply:
Added an issue for this: https://github.com/arcus-azure/arcus.backgroundjobs/issues/140